content/changelog/2024-02-02-images-update.md

```diff
@@ -0,0 +1,35 @@
+---
+title: February 2024 images update (part 1)
+date: 2024-02-02
+tags:
+  - images
+  - update
+  - security
+authors:
+  - name: David Legrand
+    link: https://github.com/davlgd
+    image: https://github.com/davlgd.png?size=40
+description: An in-depth work leading to faster updates
+excludeSearch: true
+---
+
+Over the past few months, we've overhauled the way we build our applications and add-ons images from tooling to included binaries. We'll deliver soon the first public milestone of this project, which will leads us to more frequent releases.
```

Comment:
Remove ` We'll deliver soon the first public milestone of this project, which will leads us to more frequent releases` (no future specs or announcements, just documenting what is effective at the moment of publication). Maybe add `This is the first part of the updated images`, or something like that, instead?

Reply:
Ok then, it's not doc do let's go